src/apps/loyalty/containers/login/index.tsx

```diff
@@ -92,11 +92,17 @@ class Login extends React.Component<LoginProps, LoginState> {
   }
 
   handleEmailChange (e) {
-    this.setState({email: e.target.value})
+    this.setState({
+      email: e.target.value,
+      error: "",
```

Comment:
@alloy, I didn't think of that. Do you prefer `null` over an empty string but because it makes more sense syntax wise? Don't they both evaluate to falsey?